DDDEastAnglia.Tests/SponsorModelQueryBuilder.cs

```diff
@@ -0,0 +1,15 @@
+using DDDEastAnglia.Controllers;
+using DDDEastAnglia.DataAccess;
+
+namespace DDDEastAnglia.Tests
+{
+    public class SponsorModelQueryBuilder
```

Comment:
At the moment it's only used in the `HomeController` so If I replace the `HomeControllerBuilder` with a simple method then all this goes away for now..